### DISCUSSION TITLE
/spawn list keeps saying nothing can spawn on an ordinary block

### DISCUSSION THREAD

**Discussion by integerrr** [gnembon/fabric-carpet#1379] [Q&A] [answered]
which seems wrong to me, it also keeps saying the maximum spawn Y value is 1 block above *every* block i checked. Did i somehow got some kind of config wrong? Any help is greatly appreciated

<img width="665" alt="image" src="https://user-images.githubusercontent.com/7279624/160650553-952db042-0071-4cac-8711-7a8c62668515.png">

**Comment by replaceitem** [comment, marked as answer]
Are you entering the position of the block, or the block of air above it?
If you are using tab autocompletion, it will fill the position of the block, but you need to enter the position above it, since that's where mobs would spawn.

**Reply by integerrr** [reply]
yeah my bad i was being stupid, i was indeed tab autocompleting, thanks!
